config/openzwave/.gitignore

```diff
@@ -0,0 +1,4 @@
+# This directory will be populated by openzwave
+
+*
+!.gitignore
```

Comment:
this is where the openwave adapter outputs logs and the network persistent XML file. This is also where we can put preconfigured config files.